introduction/support.md

```diff
@@ -53,33 +49,33 @@ No worries! Here's how we recommend proceeding:
    archives](https://www.mail-archive.com/qubes-users@googlegroups.com/). There
    may have already been a relevant thread.
 
-### I didn't find a solution or an answer!
+### Ask the community for help
 
-Sorry to hear that! In that case, we recommend asking for help on the [Qubes
+If you are not able to find an answer in our docs or community archives, we recommend asking for help on the [Qubes
 Forum](https://forum.qubes-os.org/) or on the [`qubes-users` mailing
 list](#qubes-users). Choose the venue you prefer, but please don't ask on both
-at the same time! Before you ask, please review our [discussion
+at the same time. Before you ask, please review our [discussion
 guidelines](#discussion-guidelines) and StackOverflow's advice on [how to ask
 good questions](https://stackoverflow.com/help/how-to-ask). Don't forget to
 [stay safe](#staying-safe)!
 
-### I don't need support, but I think I found a bug
+### Report a bug
 
 We'd be grateful if you reported it (but please make sure no one else has
 already reported it first)! Please see [Issue Tracking](/doc/issue-tracking/)
 for details.
 
-### I don't need support, but I'd like to request a feature
+### Report a security vulnerability
 
-No promises, but we'd be happy to consider it! Please see [Issue
-Tracking](/doc/issue-tracking/) for details.
+That sounds more like you helping us! Thanks! Please see [Reporting Security
+Issues in Qubes OS](/security/#reporting-security-issues-in-qubes-os).
 
-### I just want to discuss Qubes!
+### Request a feature
 
-Great! Head on over to the [Qubes Forum](https://forum.qubes-os.org/) or
-the [`qubes-users` mailing list](#qubes-users).
```

Comment:
What's the rationale for eliminating this section? There have been many times in the past when people have asked where they should go to discuss Qubes, especially after being told that qubes-issues is *not* that place.

Reply:
I don't remembering doing this, tbh. "I just want to discuss Qubes!" is a strange header. The exclamation-point in a header also feels extraneous.

Reply:
> I don't remembering doing this, tbh. "I just want to discuss Qubes!" is a strange header. The exclamation-point in a header also feels extraneous. 

Agreed, but I couldn't think of anything better. ☹️